### PR TITLE
Issue/bug template changed to a modern template

### PR DESCRIPTION
**Pull request by sadrasabouri**
### Added
- `feature_request.yml` template
- `config.yml` for issue template
### Changed
- Bug report template modified

### CODE REVIEW

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Target branch is `main`

**Comment by sadrasabouri** [comment]
> @sadrasabouri Target branch is `main`

Sorry, fixed.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  - type: dropdown
+    id: mytimer-version
+    attributes:
+      label: MyTimer version
+      description: Which version of MyTimer are you using?
+      options:
+        - MyTimer 1.1
+        - MyTimer 1.0
+        - MyTimer 0.9
+        - MyTimer 0.7
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/bug_report.yml:87]: MyTimer 0.8

File: otherfiles/RELEASE.md [hunk 2]
```diff
@@ -12,7 +12,9 @@
 	1. Add a new header under `Unreleased` section (Example: `## [0.1] - 2022-08-17`)
 	2. Add a new compare link to the end of the file (Example: `[0.2]: https://github.com/sepandhaghighi/mytimer/compare/v0.1...v0.2`)
 	3. Update `dev` compare link (Example: `[Unreleased]: https://github.com/sepandhaghighi/mytimer/compare/v0.2...dev`)
-4. Create a PR from `release` to `dev`
+4. Update `./github/ISSUE_TEMPLATE/bug_report.yml`
```
Comment (sepandhaghighi) [inline otherfiles/RELEASE.md:15]: `/.github/ISSUE_TEMPLATE/bug_report.yml`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Please take a look at my comments 🔥

File: otherfiles/RELEASE.md (hunk 2 above)
Comment (sadrasabouri) [reply otherfiles/RELEASE.md]: fixed in [6917646](https://github.com/sepandhaghighi/mytimer/pull/80/commits/6917646dcf94b3d4d9cc059949bc17adf3ab95ef).

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (sadrasabouri) [reply .github/ISSUE_TEMPLATE/bug_report.yml:88]: fixed in [6917646](https://github.com/sepandhaghighi/mytimer/pull/80/commits/6917646dcf94b3d4d9cc059949bc17adf3ab95ef).

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Sometimes, you use `>` in the placeholder section of certain repositories, while other times you don't.
Are you sure that you are using the correct template?

**Comment by sadrasabouri** [comment]
> @sadrasabouri Sometimes, you use `>` in the placeholder section of certain repositories, while other times you don't. Are you sure that you are using the correct template?

Hey Sepand, thanks for reviewing. I figured the problem. Apparently, I copied the issue template from a un-pulled repository (`pycm`) on my local computer. I double-check all other repositories and they are OK, including `art`.
The point is that only `textarea` element had that bug, so for other element we are fine and don't need `>`. I will update this PR soon after this comment.

**Review by sepandhaghighi — APPROVED** [review]
Well done 🔥

**Pull request merged**